apps/nextjs/package.json

```diff
@@ -18,6 +19,8 @@
     "test": "pnpm with-env jest --colors --config jest.config.js",
     "test-e2e": "pnpm with-env playwright test",
     "test-e2e-ui": "pnpm with-env playwright test --ui",
+    "test-e2e-ui-serve": "pnpm build && pnpm start --port 4848 --turbo",
```

Comment:
Attempting to run E2E tests on a built version of the app so that they are less flaky

Reply:
That could be very cool